endpoints-framework/src/main/java/com/google/api/server/spi/EndpointMethod.java

```diff
@@ -198,25 +214,30 @@ private void validateNoWildcards(Type[] types) {
     }
   }
 
+  private boolean isWildcardType(Type type) {
```

Comment:
Moved to the Types class and added some tests. This will be used elsewhere in the future.